ccx_messaging/publishers/idp_publisher.py

```diff
@@ -0,0 +1,39 @@
+"""Module implementing a IDP publisher to Kafka topic."""
+import json
+from typing import Dict
+import logging
+from ccx_messaging.error import CCXMessagingError
+from ccx_messaging.publishers.kafka_publisher import KafkaPublisher
+LOG = logging.getLogger(__name__)
+
+class IDPPublisher(KafkaPublisher):
+
+    """Publisher for interanal data pipeline."""
+
+    def publish(self, input_msg: Dict, report: str) -> None:
+        """Publish response as Kafka message to outgoing topic."""
+        output_msg = {}
+        try:
+            output_msg = json.loads(report)
+        except (TypeError, json.decoder.JSONDecodeError) as err:
+            raise CCXMessagingError("Could not parse report; report is not in JSON format") from err
+        output_msg.pop("reports", None)
```

Comment:
why?

Reply:
Just a check if the report was loaded propperly i can remove it from there if you think is not needed

Reply:
Ok i removed it from there